docs/contributing/managing-pull-requests.md

```diff
@@ -140,6 +142,18 @@ These are bad PR titles because they are generic, don't communicate the change p
 - Be objective and limit nitpicks (a few are fine if they add value or improve code readability)
 - Don't suggest and expect changes out of scope which are best addressed in a separate PR
 
+## Pushing changes to a remote fork
+
+Sometimes the easiest way to unblock a stalled PR is to sort out merge conflicts or apply remaining suggestions. When the GitHub UI won't cut it, you can (often) apply changes directly to someone's remote fork with Git:
+
+- Add their Gatsby fork as a remote:<br />`git add <forkname> git@github.com:<username>/gatsby.git`
```

Comment:
```suggestion
- Add their Gatsby fork as a remote:<br />`git remote add <forkname> git@github.com:<username>/gatsby.git`
```
I think this needs the remote keyword? Correct me if I'm wrong

Reply:
You are correct! I accidentally'd a word. 😅